### PR TITLE
Bug 1555336 Edited info for nm_controlled

### PR DESCRIPTION
**Pull request by bfallonf**
For:
https://bugzilla.redhat.com/show_bug.cgi?id=1555336

Will verify in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
Thanks @dcbw . @openshift/team-documentation PTAL

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -452,11 +452,17 @@ nodes.
 ==== NetworkManager
 
 //tag::networkmanager[]
-*NetworkManager*, a program for providing detection and configuration for systems to automatically connect to the network, is required on the nodes in
-order to populate *dnsmasq* with the DNS IP addresses. DNS does not work
-properly when the network interface for {product-title} has `NM_CONTROLLED=no`.
+*NetworkManager*, a program for providing detection and configuration for
+systems to automatically connect to the network, is required on the nodes in
```
Comment (gaurav-nelson) [inline install_config/install/prerequisites.adoc]: maybe s/network, is/network. It is

Comment (gaurav-nelson) [reply install_config/install/prerequisites.adoc]: s/in order to/to

File: install_config/install/prerequisites.adoc [hunk 2]
```diff
@@ -452,11 +452,17 @@ nodes.
...
+order to populate *dnsmasq* with the DNS IP addresses. 
+
+`NM_CONTROLLED` is set to `yes` by default. If `NM_CONTROLLED` is set to `no`,
```
Comment (gaurav-nelson) [inline install_config/install/prerequisites.adoc]: I would rewrite this as
```
`NM_CONTROLLED` is set to `yes` by default. If `NM_CONTROLLED` is set to `no`,
the NetworkManager dispatch script does not create the relevant
*_origin-upstream-dns.conf_* dnsmasq file. For such cases, you are required to
configure *dnsmasq* manually.
```

**Review by gaurav-nelson — APPROVED** [review]
Few nits with one suggestions. Rest LGTM 🏆

**Comment by bfallonf** [comment]
Thanks @gaurav-nelson . I put in some of your points.

I'll merge.

**Pull request merged**